crates/release_plz_core/src/next_ver.rs

```diff
@@ -578,6 +594,18 @@ fn get_changelog(
     Ok(changelog)
 }
 
+fn get_package_path(
+    package: &Package,
+    repository: &Repo,
+    project_root: &Path,
+) -> PathBuf {
```

Comment:
```suggestion
) -> anyhow::Result<PathBuf> {
```
Let's return a Result, so that we don't panic in the unwraps.